R/obtain_emiss.R

```diff
@@ -117,8 +119,9 @@ obtain_emiss <- function(object, level = "group", burn_in = NULL){
       if(n_cat > 0){
         for(q in 1:n_cat){
           ind <- which_cat[q]
-          est[[ind]][] <- matrix(round(apply(object$emiss_prob_bar[[q]][((burn_in + 1): J),], 2, median),3),
-                               byrow = TRUE, ncol = q_emiss[ind], nrow = m)
+          emiss_int_est[[ind]][] <- matrix(apply(object$emiss_int_bar[[j]][((burn_in + 1): J),], 2, median), #not sure if I should do anything to it
```

Comment:
the 'j' in object$emiss_int_bar[[j]] should be a q i think, given the loop? Or ind?